File: Web/Client/src/app/models/http-models/scanned-student.ts
import { Lane } from "../lane.model";

export class ScannedStudent {
  flightNumber: number;
  cardId: number;
  studentId: number;
  name: string;
  lane: Lane;
  parentLicense: string;
  parentName: string;
  parentId: number;
  teacherId: number;
}

File: WebAPI/Helpers/HubModels/ScannedStudent.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using WebAPI.Data;

namespace WebAPI.Helpers.HubModels
{
    public class ScannedStudent
    {
        public int FlightNumber { get; set; }
        public int CardId { get; set; }
        public int StudentId { get; set; }
        public string Name { get; set; }
        public Lane Lane { get; set; }
        public string ParentLicense { get; set; }
        public int ParentId { get; set; }
        public string ParentName { get; set; }
        public int FligthId { get; set; }
        public int? TeacherId { get; set; }

    }
}
